### PR TITLE
refactor calculateRank.js

### PR DESCRIPTION
**Pull request by kawarimidoll**
Hello, let me propose some refactorings.

- use `else if` instead of duplicated conditions
- use `const` instead of `var`
- simplify expressions

Thank you for maintaining such a wonderful project.

### CODE REVIEW

**Review by filiptronicek — APPROVED** [review]
LGTM ✅

File: src/calculateRank.js [hunk 1]
```diff
@@ -67,23 +64,13 @@ function calculateRank({
...
+  } else if (normalizedScore < RANK_A2_VALUE) {
     level = "A++";
-  }
-  if (normalizedScore >= RANK_A2_VALUE && normalizedScore < RANK_A3_VALUE) {
+  } else if (normalizedScore < RANK_A3_VALUE) {
     level = "A+";
-  }
-  if (normalizedScore >= RANK_A3_VALUE && normalizedScore < RANK_B_VALUE) {
+  } else if (normalizedScore < RANK_B_VALUE) {
     level = "B+";
```
Comment (Plavit) [inline src/calculateRank.js:74]: Was the B+ intended to be the lowest rank available? I thought there used to be A

**Review by Plavit — APPROVED** [review]
Looks good as refactoring, though the raking still makes little sense

File: src/calculateRank.js (hunk 1 above)
Comment (kawarimidoll) [reply src/calculateRank.js:74]: Yes, the B+ is the lowest and the A is not available now. There was the rank A in the first version, which has been changed to B+.
(please check the [history for src/calculateRank.js](https://github.com/anuraghazra/github-readme-stats/commits/master/src/calculateRank.js))

This PR is simply a refactoring, so the ranking algorithm has not been changed.

**Pull request closed without merging**